x11-wm/stumpwm/stumpwm-22.05.ebuild

```
@@ -0,0 +1,76 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit common-lisp-3 autotools desktop elisp-common xdg-utils
+
+DESCRIPTION="Stumpwm is a Window Manager written entirely in Common Lisp"
```

```suggestion
DESCRIPTION="A Window Manager written entirely in Common Lisp"
```
or
```suggestion
DESCRIPTION="Window Manager written entirely in Common Lisp"
```
We don't repeat `PN` here.